test/mysql/index.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const mysql = require("mysql");
+
+function main() {
+  // Since we don't use a real database, Connection.query will thow
+  // an error or pass the err object to the provided callback.
+  // We want to record the return event, too, so we pass an indicator
+  // to our mysql hook to make it record the return event even if there
+  // is an error.
+  const APP_MAP_TEST = "__AppMapTest";
```

Comment:
I don't like this — the test scripts should run the same when ran without appmap, this breaks this expectation.